canonical/snapcraft.io
# Move model overview CTA and update title

## Done
- Moved the CTA on model overview to the top of the view
- Added a breadcrumb navigation
- Hide h1 except for screen readers

## How to QA
- Go to https://snapcraft-io-4373.demos.haus/admin/njwQYXFnS7ppo7LaGxoh7aqVZc1CPi26/models/aliomt-test-model-no-api-key1
- Check that the CTA is at the top of the form
- Check that there is a breadcrumb navigation
- Check that the h1 is not visible

## Issue / Card
Fixes https://warthogs.atlassian.net/browse/WD-5631
